webpack.common.config.js

```diff
@@ -129,6 +129,9 @@ var resolve = {
         'diffTool': staticPath('js/diffTool.js'),
         'mathrender': staticPath('js/mathrender.js'),
         'citations': staticPath('js/citations.js'),
+        'jstz': staticPath('vendor/bower_components/jsTimezoneDetect/index.js'),
+        'addonNodeConfig': staticPath('js/addonNodeConfig.js'),
```

Comment:
These files are part of https://github.com/CenterForOpenScience/osf.io/pull/2014, and probably shouldn't be a part of this PR.